CesiumNativeTests/CMakeLists.txt

```diff
@@ -4,9 +4,8 @@ configure_cesium_library(cesium-native-tests)
 # Add tests here, ensure they define the TEST_SOURCES / TEST_HEADERS
 # properties.
 set(cesium_native_targets
+    Cesium3DTilesReader
     Cesium3DTilesSelection
-    CesiumAsync
-    CesiumGeometry
```

Comment:
I'm guessing these were removed accidentally. I'll add them back.

Reply:
Yeah that was a bad merge on my part, thanks.